packages/bot-utils/src/help.ts

```
@@ -655,8 +730,8 @@ ${codeBlock(authHelpTexts.commands.reduce((s, cmd, i) => {
 		const spaces = ' '.repeat(maxAuthCmdLength - cmd.name.length);
 		return (
 			s
-			+ `  ${cmd.name}${spaces}${cmd.short}`
-			+ (i < authHelpTexts.commands.length - 1 ? '\n' : '')
+				+ `  ${cmd.name}${spaces}${cmd.shortDescription}`
+				+ (i < authHelpTexts.commands.length - 1 ? '\n' : '')
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)